src/main/kotlin/com/lwise/util/CatApiClient.kt

```diff
@@ -0,0 +1,53 @@
+package com.lwise.util
+
+import io.github.cdimascio.dotenv.dotenv
+import okhttp3.HttpUrl
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import org.json.JSONArray
+import java.io.IOException
+
+object CatApiClient {
+
+    val dotenv = dotenv {
+        ignoreIfMissing = true
+    }
+
+    val API_KEY = dotenv["DATABASE_URL"]

```

Comment:
Why is the `API_KEY` using `DATABASE_URL`?

Reply:
because I goofed

Reply:
fixed